src/spreadforcreater/SpreadforCreator.styled.js

```diff
@@ -0,0 +1,182 @@
+import styled from "styled-components";
+
+export const CreatorSection = styled.div`
+  background-color: #fff6f6;
+  max-height: 525px;
+  align-items: flex-end;
+  display: flex;
+  justify-content: center;
+  @media (max-width: 700px) {
+    align-items: baseline;
+  }
+`;
+
+export const CreatorWrapper = styled.div`
+  background-color: #fff6f6;
+  height: fit-content;
+  width: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 0px 60px 0px 0px;
+  @media (max-width: 700px) {
+    padding: 0px 0px 0px 0px;
+  }
+`;
+
+export const LeftImageDiv = styled.div`
+  max-width: 48%;
+  width: 100%;
+  position: relative;
+`;
+export const CreatorLeftSideContainer = styled.div`
+  max-width: 50%;
+  width: 100%;
+  position: relative;
+  @media (max-width: 700px) {
+    max-width: 100%;
+  }
+`;
+export const CreatorRightSideContainer = styled.div`
+  max-width: 50%;
+  width: 100%;
+  position: relative;
+  display: flex;
+  padding: 66px 0px 8px 4px;
+  flex-direction: column;
+  @media (max-width: 700px) {
+    max-width: 100%;
+  }
+`;
+export const CreatorImage = styled.img`
+  max-width: 394px;
+  width: 82%;
```

Comment:
when you give max width than width should be 100% @PRABgit01

Reply:
fixed it